route4me-csharp-sdk/Route4MeSDKLibrary/Route4MeManagerV5.cs

```diff
@@ -1015,12 +1028,16 @@ public ResultResponse BulkCreateTeamMembers(TeamRequest[] membersParams, out Res
                 }));
             }
 
-            return GetJsonObjectFromAPIAsync<DriverReview>(parameters,
+            var result = GetJsonObjectFromAPIAsync<DriverReview>(parameters,
                 R4MEInfrastructureSettingsV5.DriverReview + "/" + parameters.RatingId,
                 HttpMethodType.Get,
                 null,
                 true,
                 false);
+
+            return Task.FromResult(
```

Comment:
What is the purpose of this change? I would recommend to avoid unnecessary memory traffic here. You can just return result or revert changes.

Reply:
See the comment above for similar question.